packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/textFeatures.ts

```diff
@@ -87,6 +94,36 @@ const OutdentWhenTabText: BuildInEditFeature<PluginKeyboardEvent> = {
     },
 };
 
+/**
+ * Requires @see ExperimentalFeatures.TabKeyTextFeatures to be enabled also,
+ *          Assign a getter function to the getCustomData With the Key: FocusNextElementKey, to retrieve the element that
+ *          Should be the next focus
+ * When press Alt + F10, move the focus to the next element provided by getCustom Data with Key: FocusNextElement
+ */
+const FocusNextElement: BuildInEditFeature<PluginKeyboardEvent> = {
+    keys: [Keys.F10],
+    shouldHandleEvent: (event, editor) => {
+        return (
+            editor.isFeatureEnabled(ExperimentalFeatures.TabKeyTextFeatures) &&
+            event.rawEvent.altKey &&
+            cacheGetEventData(event, FOCUS_NEXT_ELEMENT_KEY, () =>
+                editor.getCustomData(FOCUS_NEXT_ELEMENT_KEY)
+            )
+        );
+    },
+    handleEvent: (event, editor) => {
+        const element = cacheGetEventData(
+            event,
+            FOCUS_NEXT_ELEMENT_KEY,
+            editor.getCustomData(FOCUS_NEXT_ELEMENT_KEY)
```

Comment:
This is too tricky, looks doesn't like a formal API.

And the next focusable control can change during editing, so that means caller need to monitor that change and update to editor all the time.

I strongly recommend we don't do this in rooster, but do it in a plugin in OWA instead.

If we really want to do in rooster, it should be an event and still let caller handle the event to do focus.

Reply:
Sounds good, and if a user wants to use the TabKeyFeatures, we can just mention to add this functionality to their current implementation.